docs/content/en/docs/getting-started/enforcement.md

```diff
@@ -1,94 +1,120 @@
 ---
 title: "Policy Enforcement"
 weight: 6
-description: "Policy Enforcement"
+description: "Enforcing restrictions with Tetragon"
 ---
 
-This adds a network and file policy enforcement on top of execution, file tracing
-and networking policy already deployed in the quick start.  In this use case we use
-a namespace filter to limit the scope of the enforcement policy to just the `default`
-namespace we installed the demo application in from the
-[Quick Kubernetes Install]({{< ref "docs/getting-started/install-k8s" >}}).
+Tetragon's tracing policies support monitoring events, such as file access
+events or network connection events, as well as enforcing restrictions on those
+events. Using in-kernel filtering in Tetragon provides a key performance
```

Comment:
so Tetragon doesn't really "enforce restrictions on events", it enforces restriction directly on the kernel functions. The events are just a by-product of Tetragon. Maybe the wording could be changed here?